sponge/src/main/java/net/buycraft/plugin/sponge/signs/purchases/RecentPurchaseSignPosition.java

```diff
@@ -0,0 +1,16 @@
+package net.buycraft.plugin.sponge.signs.purchases;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.spongepowered.api.world.Location;
+import org.spongepowered.api.world.World;
+
+@Value
+@AllArgsConstructor
+public class RecentPurchaseSignPosition {
+
+    private final Location<World> location;
```

Comment:
No clue. Was going to implement a different method anyways, using Sponge’s Serializable.
> On Mar 2, 2016, at 10:28 PM, Tux <notifications@github.com> wrote:
> 
> In sponge/src/main/java/net/buycraft/plugin/sponge/signs/purchases/RecentPurchaseSignPosition.java <https://github.com/BuycraftPlugin/BuycraftX/pull/19#discussion_r54835023>:
> 
> > @@ -0,0 +1,16 @@
> > +package net.buycraft.plugin.sponge.signs.purchases;
> > +
> > +import lombok.AllArgsConstructor;
> > +import lombok.Value;
> > +import org.spongepowered.api.world.Location;
> > +import org.spongepowered.api.world.World;
> > +
> > +@Value
> > +@AllArgsConstructor
> > +public class RecentPurchaseSignPosition {
> > +
> > +    private final Location<World> location;
> Is this safe to serialize with Gson?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/BuycraftPlugin/BuycraftX/pull/19/files#r54835023>.
>